src/Symfony/Component/Console/Descriptor/XmlDescriptor.php

```diff
@@ -201,32 +205,61 @@ private function getInputOptionDocument(InputOption $option): \DOMDocument
         $objectXML->setAttribute('accept_value', $option->acceptValue() ? 1 : 0);
         $objectXML->setAttribute('is_value_required', $option->isValueRequired() ? 1 : 0);
         $objectXML->setAttribute('is_multiple', $option->isArray() ? 1 : 0);
-        $objectXML->appendChild($descriptionXML = $dom->createElement('description'));
+        $objectXML->appendChild($descriptionXML = $this->createElement($dom, 'description'));
         $descriptionXML->appendChild($dom->createTextNode($option->getDescription()));
 
         if ($option->acceptValue()) {
             $defaults = \is_array($option->getDefault()) ? $option->getDefault() : (\is_bool($option->getDefault()) ? [var_export($option->getDefault(), true)] : ($option->getDefault() ? [$option->getDefault()] : []));
-            $objectXML->appendChild($defaultsXML = $dom->createElement('defaults'));
+            $objectXML->appendChild($defaultsXML = $this->createElement($dom, 'defaults'));
 
             if (!empty($defaults)) {
                 foreach ($defaults as $default) {
-                    $defaultsXML->appendChild($defaultXML = $dom->createElement('default'));
+                    $defaultsXML->appendChild($defaultXML = $this->createElement($dom, 'default'));
                     $defaultXML->appendChild($dom->createTextNode($default));
                 }
             }
         }
 
-        if ($option->isNegatable()) {
-            $dom->appendChild($objectXML = $dom->createElement('option'));
-            $objectXML->setAttribute('name', '--no-'.$option->getName());
-            $objectXML->setAttribute('shortcut', '');
-            $objectXML->setAttribute('accept_value', 0);
-            $objectXML->setAttribute('is_value_required', 0);
-            $objectXML->setAttribute('is_multiple', 0);
-            $objectXML->appendChild($descriptionXML = $dom->createElement('description'));
-            $descriptionXML->appendChild($dom->createTextNode('Negate the "--'.$option->getName().'" option'));
-        }
+        return $dom;
+    }
+
+    private function getNegatableInputOptionDocument(InputOption $option): \DOMDocument|\DOM\XMLDocument
```

Comment:
Necessary to create this new method. Otherwise, when the command option is negatable, `getInputOptionDocument()` return a document with two child nodes, which is not permitted by the new ext-dom.